.circleci/config.yml

```diff
@@ -42,7 +46,7 @@ jobs:
     steps:
       - checkout
       - run: ./scripts/run_in_ci.sh echo 'Building image!'
-      - run: docker login --username $DOCKER_ID --password $DOCKER_PASSWD
+      - run: docker login --username DOCKERID−−passwordDOCKER_PASSWD
```

Comment:
Did you mean to change this?

Reply:
no ... not sure how that happened.

Reply:
I think everything is all set now, let me know if there's anything else you have questions on regarding the analysis.